### ISSUE TITLE
Mega Ind V1.1 CAN protocol

### ISSUE THREAD

**Issue by Maudal78** [SequentMicrosystems/megaind-rpi#2]
Dear, 
are there any writed  python CAN lib for MegaInd to comunicate with CAN devices?
Were can I find?
Any example?
Thanks
Regards

**Comment by alexburcea2877** [comment]
Hi,

The current firmware version does not support CAN, A new firmware version
and python lib to support CAN is under construction.
Regards

În joi, 26 nov. 2020 la 11:22, maudal78 <notifications@github.com> a scris:

**Comment by Sebastien-GG** [comment]
Dear Alex
Do you plan to release soon new Firmware with CAN bus update ?
Thanks
Regards

**Comment by alexburcea2877** [comment]
Hi,

We don't have the time for finishing the implementation but I think in a month or so we will have the first version.

Regards

**Comment by Sebastien-GG** [comment]
Dear Alex 
In the event of you worked on CAN bus update, i m still interested in it !
Regards

**Comment by Sebastien-GG** [comment]
Dear ALex
Have you had time to do Can bus firmware update ?
Regrads

**Comment by alexburcea2877** [comment]
Not yet!

În joi, 2 sept. 2021 la 12:48, Sebastien-GG ***@***.***> a
scris:

**Comment by JarrodJackson98** [comment]
Did you ever get round to this?

**Comment by alexburcea2877** [comment]
No, and because we run into some problems, I don't think we will support CAN on these old cards, sorry.
